### ISSUE TITLE
CLI: nested `contrib.telegram`/`discord` broken in v4.49.0

### ISSUE THREAD

**Issue by casperdcl** [tqdm/tqdm#1074]
visual output bug (found by @JulienMaille https://github.com/tqdm/tqdm/pull/949#issuecomment-725336816)

`tqdm==4.48.2` works fine, `tqdm==4.49.0` doesn't:

```python
from tqdm import trange
from tqdm.contrib.telegram import ttgrange


for i in ttgrange(100):
    for j in trange(int(1e6), leave=False):  # should automatically be at position=1, not 0
        pass
```

Likely other CLI submodules may be broken by `v4.49.0` too?

**Comment by JulienMaille** [comment]
Could it have something to do with issue https://github.com/tqdm/tqdm/issues/509 ?

**Comment by casperdcl** [comment]
Yes it is; I just haven't had time to push a fix yet :)

**Comment by casperdcl** [comment]
ugh; was bound to miss one... will be fixed in next release (`tqdm>=4.52.1`, #1075)

![image](https://user-images.githubusercontent.com/10780059/99858614-d8a0e580-2b85-11eb-808f-aba764b9513c.png)
